### PR TITLE
support for handling multiple file uploads

### PR DESCRIPTION
**Pull request by benschinn**
It's not pretty, but it's working. Feedback welcome. We may also want to set a limit on number of files.

currently payload looks like:
```
[{"caseNumber": "123" ...}, {"caseNumber": "124" ...}]
```
should we change the payload to have some indication of what parsed data belongs to which docket? something like this? 
```
[{"fileName": "docket.pdf", "caseNumber": "123" ...}, {"fileName": "docket_1.pdf", "caseNumber": "124" ...}]
```

### CODE REVIEW

File: backend/apis/parse-docket-pdf.api.js [hunk 1]
```diff
@@ -6,8 +6,10 @@ const { generateFlagJson } = require("./generate-flag-json.utils");
 
 app.post("/api/docket-pdfs", (req, res) => {
   let busboy;
+  const busBoyConfig = { headers: req.headers };
```
Comment (benschinn) [inline backend/apis/parse-docket-pdf.api.js]: Here we can add additional configuration such as limiting number of files. Do we know the max number of dockets a user can upload?

File: backend/apis/parse-docket-pdf.api.js [hunk 2]
```diff
@@ -25,6 +29,7 @@ app.post("/api/docket-pdfs", (req, res) => {
         .send({ error: "Uploaded file did not have mimetype application/pdf" });
     }
     fileWasUploaded = true;
+    if (filename) count++;
```
Comment (benschinn) [inline backend/apis/parse-docket-pdf.api.js]: busboy processes one file at a time. this is how I track the number of files were uploaded. I think messing with counters can be error prone. if anyone has a better solution, please share!

File: backend/apis/parse-docket-pdf.api.js [hunk 3]
```diff
@@ -34,9 +39,13 @@ app.post("/api/docket-pdfs", (req, res) => {
             if (processJson) {
               //comment this back in if you want to test flag json
               //res.send(generateFlagJson(parsePdfText(thePdf.text)));
-              res.send(parsePdfText(thePdf.text));
+              payload.push(parsePdfText(thePdf.text));
+              count--;
+              if (count === 0) res.send(payload);
```
Comment (benschinn) [inline backend/apis/parse-docket-pdf.api.js]: also, do we prefer using if statement here or something like this instead `count === 0 && res.send(payload)` instead?

Comment (lukeschunk) [reply backend/apis/parse-docket-pdf.api.js]: i think this is fine 👍

File: backend/apis/parse-docket-pdf.api.js (hunk 1 above)
Comment (benschinn) [reply backend/apis/parse-docket-pdf.api.js]: I think it's hard to set a limit on number of dockets. for now it is more reasonable to set limit on `fileSize`. It has been set to 10mb which should be plenty.

File: backend/apis/parse-docket-pdf.api.js (hunk 2 above)
Comment (benschinn) [reply backend/apis/parse-docket-pdf.api.js]: no feedback. will resolve for now.

**Comment by benschinn** [comment]
@lukeschunk could you merge it in? I don't have write access to the repo.

**Pull request merged**